### PR TITLE
Add example of load shed point configuration

### LINKED ISSUE #30267: Add config example on how to configure load shed point in overload manager
Load shed points [documentation](https://github.com/envoyproxy/envoy/blob/main/docs/root/configuration/operations/overload_manager/overload_manager.rst#load-shed-points) would benefit from having a config example.

### PR DESCRIPTION
**Pull request by nezdolik**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
Fixes #30267

### CODE REVIEW

**Comment by nezdolik** [comment]
cc @KBaichoo

File: docs/root/configuration/operations/overload_manager/overload_manager.rst [hunk 1]
```diff
@@ -138,6 +138,26 @@ Compared to overload actions, Load Shed Points are also more flexible to
 integrate custom (e.g. company inteneral) Load Shed Points as long as the extension
 has access to the Overload Manager to request the custom Load Shed Point.
 
+As an example, below is partial Overload Manager configuration with a core load shed point
+``envoy.load_shed_points.http_connection_manager_decode_headers`` and a maximum threshold
+for the ``envoy.resource_monitors.fixed_heap`` monitor. Once threshold is reached, Envoy
```
Comment (KBaichoo) [inline docs/root/configuration/operations/overload_manager/overload_manager.rst:143]: Once the threshold*

File: docs/root/configuration/operations/overload_manager/overload_manager.rst [hunk 2]
```diff
@@ -138,6 +138,26 @@ Compared to overload actions, Load Shed Points are also more flexible to
...
+will reject new HTTP streams by sending local reply. Monitor needs to be configured as well:
```
Comment (KBaichoo) [inline docs/root/configuration/operations/overload_manager/overload_manager.rst:144]: sending local replies. The resource monitor*

**Review by KBaichoo — COMMENTED** [review]
Thanks for adding an example, just minor changes to make it flow better

/wait

File: docs/root/configuration/operations/overload_manager/overload_manager.rst [hunk 3]
```diff
@@ -138,6 +138,26 @@ Compared to overload actions, Load Shed Points are also more flexible to
...
+
+.. code-block:: yaml
+
+  loadshed_points:
```
Comment (phlax) [inline docs/root/configuration/operations/overload_manager/overload_manager.rst:148]: could we add this as bootstrap yaml please - that way its tested prevents bitrot

Comment (phlax) [reply docs/root/configuration/operations/overload_manager/overload_manager.rst:148]: (using `literalinclude` to include it)

**Pull request closed without merging**